### DISCUSSION TITLE
Quest helper skill helper outdated (Agility)

### DISCUSSION THREAD

**Discussion by qman621** [runelite/runelite#17837] [Ideas]
The agility skill helper incorrectly says draynor village starts at lvl 10 when it should be 1 now. The skill guide is really helpful and should be expanded to other skills also.

**Comment by Felanbird** [comment]
Quest Helper is a hub plugin, as it notes at the top of the hub tab we do not maintain those.

Support for hub plugins should be directed to the author of the plugin. You can find the support link by searching for the plugin in the Plugin Hub panel and clicking the ? button on the plugin, or by right-clicking the plugin in the plugin panel and clicking the Support menu option.
